spec/javascript/pages/browse/ContactIcons.spec.js

```diff
@@ -0,0 +1,11 @@
+import Vue from 'our_vue'
```

Comment:
Even though I don't reference it directly, this seems to make the test play nice with Buefy. The test fails if I remove it, even with using `shallowMount`